includes/class-orddd-lite-process.php

```diff
@@ -624,8 +624,8 @@ public static function orddd_lite_add_delivery_date_fee( $cart ) {
 
 		$delivery_on_cart = get_option( 'orddd_delivery_date_on_cart_page' );
 		$is_cart          = is_cart();
-		$is_ajax          = is_ajax();
-
+		$is_ajax          = wp_doing_ajax();
```

Comment:
this should work for older versions of WC. Add a condition to check the WC version